Tests/TestingApps/PaywallsTester/ci_scripts/ci_pre_xcodebuild.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
Gotta make sure that I update `Xcode Cloud` to make it work with the new project.